Src/PerformanceCollector/Perf.Shared/QuickPulseTelemetryProcessor.cs

```diff
@@ -194,7 +197,7 @@ private ITelemetryDocument ConvertRequestToTelemetryDocument(RequestTelemetry re
             {
                 try
                 {
-                    // some of the requestTelemetry properties might be deffered by using RequestTrackingTelemetryModule.DisableTrackingProperties.
+                    // some of the requestTelemetry properties might be deferred by using RequestTrackingTelemetryModule.DisableTrackingProperties.
```

Comment:
is this statement stil true in the context of new Experimental Feature suport?

Reply:
I think this is still valid.
The experimental feature is used to set this property, but this is still the property that decides to defer work.